homeassistant/components/swiss_public_transport/strings.json

```diff
@@ -49,12 +49,37 @@
       }
     }
   },
+  "services": {
+    "fetch_connections": {
+      "name": "Fetch Connections",
+      "description": "Fetch a list of connections from the swiss public transport.",
+      "fields": {
+        "config_entry_id": {
+          "name": "Integration",
+          "description": "Swiss public transport integration to fetch connections for."
```

Comment:
Do we want to say "integration" or "instance" here? I don't mind whichever, but it would be nice if it was aligned across integrations.

Reply:
Switch to instance like for `mealie`

Reply:
I don't like the word `instance` as it immediately sounds more technical but it's nothing for this PR in particular.